src/paymentservice/src/test/java/ca/bc/gov/open/pssg/rsbc/dps/paymentservice/PaymentserviceApplicationTests.java

```diff
@@ -34,20 +48,50 @@ void contextLoaded() {
 		assertThat(bamboraConfiguration).isNotNull();
 	}
 	
+	
 	/**
-	 * calculateSinglePaymentTestGood - Basic testing of the CalculateSinglePayment operation (good response expected). 
+	 * calculateSinglePaymentHttpResponseTest - Basic testing of the CalculateSinglePayment operation (good response expected). 
 	 * 
 	 * @throws Exception
 	 */
 	@Test
-	void calculateSinglePaymentTestGood() throws Exception {
+	void calculateSinglePaymentHttpResponseTest() throws Exception {
 		System.out.println("Running basic http test for calculate single payment operation...");
 		String request = "/getSinglePaymentURL?transType=P&invoiceNumber=9999&totalItemsAmount=10.05&approvedPage=http://localhost:8080/crc/beanstream/dpsProcessPayment.do&declinedPage=http://localhost:8080/crc/beanstream/dpsProcessPayment.do&errorPage=http://localhost:8080/crc/beanstream/dpsProcessPayment.do&minutesToExpire=1440&ref1=0123456&ref2=ref2String&ref3=ref3String";
 		assertThat(this.restTemplate.getForObject("http://localhost:" + port + "/paymentservice" + request,
 		        String.class)
 		).contains(PaymentServiceConstants.PAYMENT_SERVICE_RESP_MSG_OK);	
 	}
 	
+	/**
+	 * calculateSinglePaymentAlgoTest - Basic testing of the Bambora Payment client, calculateSinglePaymentURL algorithm.
+	 * 
+	 * Note: this test uses the Test property source (See test\java\resources\test.properties)
+	 * 
+	 * @throws Exception
+	 */
+	@Test
+	void calculateSinglePaymentAlgoTest() throws Exception {
+		System.out.println("Running Bambora single payment algorithm test. Test MerchantId Id is " + merchantId);
```

Comment:
should we be logging this? or slap a todo on it for logging in the future?

Reply:
agree we should not print for unit test, the assert is the important, we can customize the message on assert

Reply:
Removed